nginx/fulltest/genconfig.py

```diff
@@ -99,15 +111,18 @@ def gen_conf(filename, indexbasefilename):
            f.write("    ssl_protocols       TLSv1.3;\n")
            f.write("    ssl_ecdh_curve      "+k+";\n")
            f.write("    location / {\n")
+           f.write("            ssi    on;\n")
+           f.write("            set    $oqs_alg_name \""+sig+"-"+k+"\";\n")
            f.write("            root   html;\n")
-           f.write("            index  index.html index.htm;\n")
+           f.write("            index  success.html;\n")
            f.write("    }\n\n")
-           i.write("<li><a href=https://"+TESTFQDN+":"+str(port)+">"+sig+"/"+k+" ("+str(port)+")</a></li>\n")
+           #i.write("<li><a href=https://"+TESTFQDN+":"+str(port)+">"+sig+"/"+k+" ("+str(port)+")</a></li>\n")
+           i.write("<tr><td>"+sig+"</td><td>"+k+"</td><td>"+str(port)+"</td><td><a href=https://"+TESTFQDN+":"+str(port)+">"+sig+"/"+k+"</a></td></tr>\n")
 
            f.write("}\n\n")
            port = port+1
      f.write("}\n")
-   i.write("</ul>\n")
+   i.write("</table>\n")
    i.write("</body>\n")
    i.close()
```

Comment:
You forgot to close your `<html>` tag. Horrible!!

Reply:
Utterly unacceptable. So now fixed.